test.py

```diff
@@ -1,2 +1,7 @@
 def main():
-    pass
+    pass   
```

Comment:
The code does not comply to PEP8.

*PEP8Bear, severity NORMAL, section `python`.*

The issue can be fixed by applying the following patch:

```diff
--- a/test.py
+++ b/test.py
@@ -1,5 +1,5 @@
 def main():
-    pass   
+    pass
 
 
 print(A+    b)    
```